joomla/joomla-cms
# Minor change in selector titles

#### Summary of Changes

As discussed in https://github.com/joomla/joomla-cms/pull/9227 this PR changes title in com_cache cache view, com_module, modules view and com_languagues installed view.

![image](https://cloud.githubusercontent.com/assets/9630530/13374475/a04fe2e0-dd7d-11e5-9a64-b4df035398f4.png)

![image](https://cloud.githubusercontent.com/assets/9630530/13374473/9db98270-dd7d-11e5-9650-66d8169c664e.png)

![image](https://cloud.githubusercontent.com/assets/9630530/13374478/ac95ae18-dd7d-11e5-886a-83b0a473dac7.png)
#### Testing Instructions
1. Use latest staging
2. Install this patch
3. Check if the page title changes to "(Site)" or "(Administrator)" according to the selected client in the views described above.
#### Observations

I removed the recently added language variable because i think they are not needed as we have JADMINISTRATOR and JSITE.
If not please say so.

@brianteeman, @infograf768 please test.
